resources/scripts.js

```diff
@@ -125,112 +188,362 @@ function buttonPosition(loc) {
+    placeDialogAmount.setAttribute('max', playerMoney);
+    makePlaceNumClicked = num;
+    placeDialog.showModal();
+    return;
+}
+
+class SecondaryBet {
+    constructor(point, amount) {
+        this.point = point;
+        this.amount = amount;
+    }
+}
+
+function pushSecondaryBets() {
+    if (testObjPop(secondaryBetObj) === false) {
+        for (let i = 1; i < 11; i++) {
+            if (secondaryBetObj[i] > 0) {
+                playerMoney += secondaryBetObj[i];
+                totalBets -= secondaryBetObj[i];
+                delete secondaryBetObj[i];
+            }
+        }
+    }
+    return;
+}
+
+
 function gameOver() {
-    displayMessage(`You are bankrupt! Please choose 'New Game' from the menu to play again`);
-    buttonPosition(1);
-    rollButtonState(false);
-    betButtonState(false);
+    displayMessage(BANKRUPT);
+    playerRoll = 1;
+    buttonPosition(playerRoll);
+    buttonStates();
+    return;
+}
+
+async function moneyChange(newBet) {
+    totalBets += newBet;
+    playerMoneyElement.innerHTML = playerMoney;
+    playerBetElement.innerHTML = totalBets;
+    if (playerMoney === lastPlayerMoney) {
+        return;
+    }
+    if (playerMoney < lastPlayerMoney) {
+        playerMoneyElement.style.transition = 'all 0.5s';
+        playerMoneyElement.style.fontSize = '75%';
+        playerMoneyElement.style.color = '#ff0000';
+        await new Promise(resolve => setTimeout(resolve, 500));
+        playerMoneyElement.style.transition = 'all 0.5s';
+        playerMoneyElement.style.fontSize = '120%';
+        playerMoneyElement.style.color = '#00ff00';
+        return;
+    }
+    if (playerMoney > lastPlayerMoney) {
+        playerMoneyElement.style.transition = 'all 0.5s';
+        playerMoneyElement.style.fontSize = '175%';
+        playerMoneyElement.style.color = '#0000ff';
+        await new Promise(resolve => setTimeout(resolve, 500));
+        playerMoneyElement.style.transition = 'all 0.5s';
+        playerMoneyElement.style.fontSize = '120%';
+        playerMoneyElement.style.color = '#00ff00';
+        return;
+    }
+    return;
+}
+
+function buildSummary(msg, rolled) {
+    if (rollSummary === null) {
+        rollSummary = msg;
+        console.log(rollSummary);
+    } else {
+        rollSummary += `, ${msg}`;
+        console.log(rollSummary);
+    }
+    if (msg === SECONDARYLOSE) {
+        // catalog all lost bets
+        rollSummary += rolled;
+    }
+    return;
 }
+const SECONDARYLOSE = 'you lost your bet on';
+const PLACEWIN = 'placeholder';
+const COMEWIN = 'placeholder';
+const COMELOSE = 'placeholder';
+const COMESET = 'place set on';
 
 function displayMessage(str) {
-    document.getElementById('current-message').innerHTML = str;
+    if (rollSummary === null) {
+        document.getElementById('current-message').innerHTML = str;
+    } else if (str === NOACTION) {
+        //document.getElementById('current-message').innerHTML = `${SECONDARYNOACTION} ${rollSummary}`;
```

Comment:
Since this is commented out, perhaps it can be deleted?

Reply:
Removed commented line as this function will be refactored in next PR anyway